caseywalker/Finance-Planner
# Adding Setup Framework

# Initial setup for Finance Planner

## Description
This is the initial setup for our React app. Connecting API keys and firebase database, user authentication, react-router, and sample data. 

## Related Issue
Fixes #1 
Fixes #2,
Fixes #3,
Fixes #4,
Fixes #6,
Fixes #5,
Fixes #10,
Fixes #49. 

## Motivation and Context
We need a framework for our app to be able to add components, routes, and data from our database. This will allow us to scale out our app as we continue to add components. 

## How Can This Be Tested?
Fetch this branch, npm start, and sign in with google. Check the Components view in React Dev tools. Check that your user info is in state on App.js. The rest of the components can be found in the src files. 


## Types of changes
